src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/PackagingTest.cs

```diff
@@ -74,14 +74,33 @@ public void CheckIncludedAssemblies ([Values (false, true)] bool usesAssemblySto
 			};
 			proj.SetProperty ("AndroidUseAssemblyStore", usesAssemblyStores.ToString ());
 			proj.SetAndroidSupportedAbis ("armeabi-v7a");
+			proj.PackageReferences.Add (new Package {
+				Id = "Humanizer.Core",
+				Version = "*",
```

Comment:
We should maybe pick a version for all these for now? I could see Humanizer releasing an update and our tests could randomly start failing. `2.14.1` is on `dotnet-public` right now.